explainers/string-meta-explainer.md

```diff
@@ -0,0 +1,36 @@
+# Explainer: String-Meta
+
+The Internationalization (I18N) Working Group has been working on getting Web specifications to provide language and base direction metadata in document formats and protocols. We have a doc (String-Meta [1], see also [4]) that describes use cases and potential approaches in depth.
```

Comment:
```suggestion
The [W3C Internationalization (I18N) Working Group](https://www.w3.org/International/core/Overview) has been working on getting Web specifications to provide language and base direction metadata in document formats and protocols. We have a doc (String-Meta [1], see also [4]) that describes use cases and potential approaches in depth.
```

If this explainer is for people outside the i18n WG (for the TAG and the ECMA-402 Task Group), it might be useful to have a link to the i18n WG.